### ISSUE TITLE
Replace the Drawer with an Full Screen overlay like AetherSX2

### ISSUE THREAD

**Issue by iamnotkurtcobain** [yuzu-emu/yuzu#12937]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What feature are you suggesting?

The drawer in Yuzu is quite good but if a game drops below 60 or 30fps the drawer animation gets very laggy. That doesn't look good at all. 

It would be great if the drawer gets replaced by an overlay like AetherSX2 has. It's transparent and pauses the game currently playing. It's always smooth because there is no stupid animation.

### Why would this feature be useful?

Looks way better and would be always smooth.

**Comment by t895** [comment]
I understand where the comment is coming from but this seems primarily directed at the lag you experience when opening/closing the drawer. Replacing it with a full-screen overlay wouldn't necessarily fix the problem.

**Comment by iamnotkurtcobain** [comment]
![20240206_183912](https://github.com/yuzu-emu/yuzu/assets/88488906/53c990ea-7cd9-4e61-aece-50de3dbcb057)

The way it looks on Aethersx2

**Comment by iamnotkurtcobain** [comment]
> I understand where the comment is coming from but this seems primarily directed at the lag you experience when opening/closing the drawer. Replacing it with a full-screen overlay wouldn't necessarily fix the problem.

Yes, the opening and closing aimation is laggy. It would look better if there would be no animation at all. Or just a fade in, fade out.
